lib/internal/streams/writable.js

```diff
@@ -898,8 +908,7 @@ Writable.prototype.destroy = function(err, cb) {
 
   // Invoke pending callbacks.
   if (!state.destroyed &&
-    (state.bufferedIndex < state.buffered.length ||
-      state[kOnFinished].length)) {
+    (state.bufferedIndex < state.buffered.length || state[kOnFinished])) {
```

Comment:
Depending on the probability, I think we should change the order of this boolean expression?

Reply:
What do you syggest?